connectors/src/connectors/slack/temporal/activities.ts

```diff
@@ -559,7 +559,7 @@ export async function syncThread(
     documentUrl: sourceUrl,
     timestampMs: createdAt,
     tags,
-    parents: [],
+    parents: [channelId],
```

Comment:
Any chance we could guarantee that we use the same Id here and in the connector ressource internal Id?

I worry we change one and not the other?

Maybe it;'s not possible and we leave a comment?

Reply:
As per [slack thread](https://dust4ai.slack.com/archives/C050SM8NSPK/p1694076110181409), the convention is now to use external ids directly everywhere, so it is legit to leave it as is.

Reply:
:+1: